From 67d2b520b27a2b2cb6a1e0a8376e7c000d092fe1 Mon Sep 17 00:00:00 2001
From: "Okles-PC\\Okles" <l.oklesinski@gmail.com>
Date: Mon, 4 Dec 2017 20:59:22 +0100
Subject: [PATCH] new patch

---
 eclipse-workspace | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eclipse-workspace b/eclipse-workspace
index 219dbd3..dfd202c 160000
--- a/eclipse-workspace
+++ b/eclipse-workspace
@@ -1 +1 @@
-Subproject commit 219dbd3de472c7cb81d84dc86e46c45f68671355
+Subproject commit dfd202c49277afa029c655995fe4e1010775b871
-- 
2.14.1.windows.1

